server/src/instant/reactive/session.clj

```diff
@@ -214,7 +225,16 @@
                       :num-spam num-spam
                       :num-computations num-computations
                       :dropped-spam? drop-spam?
-                      :tx-latency-ms (e2e-tracer/tx-latency-ms (:tx-created-at event))}]
+                      :tx-latency-ms (e2e-tracer/tx-latency-ms (:tx-created-at event))}
+        {prev-attrs :session/attrs
+         version :session/versions} (rs/session store sess-id)
+        parsed-version  (some-> version (get core-version-key) (semver/parse))
+        can-skip-attrs? (and (flags/refresh-skip-attrs? app-id)
+                             parsed-version
+                             (pos? (semver/compare-semver parsed-version refresh-skip-attrs-min-version)))
+        attrs-changed?  (not= prev-attrs attrs)]
+    (when (and can-skip-attrs? attrs-changed?)
+      (rs/assoc-session! store sess-id :session/attrs attrs))
```

Comment:
Can we just store a hash of the attrs instead of storing the full set of attrs in the session?

We already have a thing that hashes in `instaql-query-reactive!`. We could either store two hashes there or store a single hash in the session and stop inluding the attrs in the hash in `instaql-query-reactive!`.